rajawali/src/main/java/org/rajawali3d/util/egl/RajawaliEGLConfigChooser.kt

```diff
@@ -0,0 +1,75 @@
+package org.rajawali3d.util.egl
+
+import android.annotation.TargetApi
+import android.opengl.EGLExt
+import android.opengl.GLSurfaceView
+import android.os.Build
+import org.rajawali3d.view.ISurface
+
+import javax.microedition.khronos.egl.EGL10
+import javax.microedition.khronos.egl.EGLConfig
+import javax.microedition.khronos.egl.EGLDisplay
+
+/**
+ * @author Jared Woolston (jwoolston@tenkiv.com)
+ */
+class RajawaliEGLConfigChooser(glMajorVersion: Int, val antiAliasingConfig: ISurface.ANTI_ALIASING_CONFIG,
+                               sampleCount: Int, bitsRed: Int, bitsGreen: Int, bitsBlue: Int, bitsAlpha: Int, bitsDepth: Int) : GLSurfaceView.EGLConfigChooser {
+
+    private val configSpec: IntArray
+
+    init {
+
+        if (this.antiAliasingConfig == ISurface.ANTI_ALIASING_CONFIG.MULTISAMPLING) {
```

Comment:
This should be a `when` statement and it can be inlined with the parameter definition.